build.gradle

```diff
@@ -562,6 +562,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind
```

Comment:
I agree that we should first break the `kafka-clients` jar, but I'm just not sure whether we should create a separate `kafka-serde-json` jar, or if we should just have a `kafka-common.jar` or finner grained `kafka-common-serialization.jar`. Since this discussion requires further discussions and go way beyond this JIRA, I would prefer to go with the first option here. cc @ewencp @ijuma